x-pack/plugins/ml/public/explorer/explorer_controller.js

```diff
@@ -87,6 +88,10 @@ module.controller('MlExplorerController', function (
   timefilter.enableTimeRangeSelector();
   timefilter.enableAutoRefreshSelector();
 
+  // Pass the timezone to the server for use when aggregating anomalies (by day / hour) for the table.
+  const tzConfig = config.get('dateFormat:tz');
+  $scope.dateFormatTz = (tzConfig !== 'Browser') ? tzConfig : moment.tz.guess();
```

Comment:
Does this need to be in `$scope` (is it used somewhere else in a template or passed on to child directives?) - otherwise could be just `const dateFormatTz = ...`?

Reply:
Agree that if this doesn't need to be shared and is only being passed around as a param then we shouldn't keep in in `$scope`. Other than that looks good. 😄